src/plugins/intel_npu/src/plugin/npuw/compiled_model.cpp

```diff
@@ -471,6 +480,293 @@ ov::npuw::CompiledModel::CompiledModel(const std::shared_ptr<ov::Model>& model,
     report_io();
 }
 
+void ov::npuw::CompiledModel::CompiledModelDesc::serialize(std::ostream& stream,
+                                                           const std::size_t& idx,
+                                                           const std::string& device) const {
+    using namespace ov::npuw::s11n;
+
+    static std::string filename = "/home/alexeysm/npuw/tmp_subs/";
```

Comment:
Caused by inability of export/import of compiled (CPU plugin for sure, NPU not tested) model handle a single stream shared with other models and meta. Can be fixed by serializing the beforementioned models into a `stringstream` and then into `stream`